requirements.txt

```diff
@@ -3,3 +3,4 @@ discord.py
 emoji
 https://github.com/Rapptz/discord.py/archive/rewrite.zip
 peewee
+Flask
```

Comment:
I don't think we need any other requirements. One can simply install Flask and work with it, right?